Sources/Filters/General/ImageMarchingSquares/index.js

```diff
@@ -11,6 +11,20 @@ const { vtkErrorMacro, vtkDebugMacro } = macro;
 // ----------------------------------------------------------------------------
 
 function vtkImageMarchingSquares(publicAPI, model) {
+  function getKernels() {
+    let kernelX = 0; // default K slicing mode
+    let kernelY = 1;
+    if (model.slicingMode === 1) {
+      kernelX = 0;
+      kernelY = 2;
+    } else if (model.slicingMode === 0) {
+      kernelX = 1;
+      kernelY = 2;
+    }
+
+    return [kernelX, kernelY];
```

Comment:
I think the following would be more efficient:
`return model.slicingMode === 2 ? [0, 1] : model.slicingMode === 1 ? [0, 2] : [1, 2];`

Reply:
The ESLint rules in vtk.js seem to not allow nested ternarys, like in the line proposed. Would a switch statement here be better? It would probably provide "better" performance, but either way this is probably just a micro-optimization.

If you really think the nested ternarys are the way to go then I can just disable the ESLint check for that line. Let me know your thoughts.

Reply:
let's leave it like this, no worries